docs/docs/cmd/planner/task/task-add.md

```diff
@@ -76,3 +85,15 @@ Adds a Microsoft Planner task with the name _My Planner Task_ for plan with the
 ```sh
 m365 planner task add --title "My Planner Task" --planId "8QZEH7b3wkSbGQobscsM5gADCBa" --bucketId "IK8tuFTwQEa5vTonM7ZMRZgAKdna" --assignedToUserNames "Allan.Carroll@contoso.com,Ida.Stevens@contoso.com" --dueDateTime "2021-12-16"
 ```
+
+Adds a Microsoft Planner task with the name _My Planner Task_ for plan with the ID _8QZEH7b3wkbGQobscsM5gADCBa_ and for the bucket with the ID _IK8tuFTwQEa5vTonM7ZMRZgAKdna_. The new task will be assigned to the users _Allan.Carroll@contoso.com_ and _Ida.Stevens@contoso.com_ who will appear first with the asssignee priority _ !!_ 
+
+```sh
+m365 planner task add --title "My Planner Task" --planId "8QZEH7b3wkSbGQobscsM5gADCBa" --bucketId "IK8tuFTwQEa5vTonM7ZMRZgAKdna" --assignedToUserNames "Allan.Carroll@contoso.com,Ida.Stevens@contoso.com" --asssigneePriority " !!"
```

Comment:
I cannot execute this example in bash, because the `!` in assigneePriority is a reserved character. I do however am able to execute it using single quotes: `' !!'`. I'm not a bash hero, so maybe everyone using it does know this including the solution. But still: should we maybe add the single quotes in the example?